backend/docker-compose.yaml

```diff
@@ -30,3 +30,52 @@ services:
     ports:
       - "7000:8000"
     command: uvicorn lm_backend.main:app --reload --workers 1 --host 0.0.0.0 --port 8000
+
+  unittest:
+    build:
+      context: .
+      dockerfile: ./Dockerfile
+    environment:
+      DATABASE_URL: postgresql+asyncpg://omnivector:rats@pgsql:5432/ratsdb
+      ARMASEC_DOMAIN: ${ARMASEC_DOMAIN}
+      ARMASEC_AUDIENCE: ${ARMASEC_AUDIENCE}
+      ARMASEC_DEBUG: ${ARMASEC_DEBUG}
+      LOG_LEVEL: ${LOG_LEVEL}
+    command: poetry run pytest
+    depends_on:
+      pgsql:
+        condition: service_healthy
+      migration:
+        condition: service_completed_successfully
+    volumes:
+      - ./tests:/app/tests
+
+  migration:
+    build:
+      context: .
+      dockerfile: ./Dockerfile
+    environment:
+      DATABASE_URL: postgresql://omnivector:rats@pgsql:5432/ratsdb
+      ARMASEC_DOMAIN: ${ARMASEC_DOMAIN}
+      ARMASEC_AUDIENCE: ${ARMASEC_AUDIENCE}
+      ARMASEC_DEBUG: ${ARMASEC_DEBUG}
+      LOG_LEVEL: ${LOG_LEVEL}
+    command: >
+      sh -c  "/usr/local/bin/pip install alembic==1.8.1 psycopg2==2.9.5 && alembic -c alembic/alembic.ini upgrade head"
```

Comment:
This also came from the [SOS API](https://github.com/omnivector-solutions/sos/blob/main/sos-api/docker-compose.yaml#L84), I updated it to use poetry, since these dependencies are already in the pyproject.toml.